xla/service/gpu/cudnn_fused_mha_rewriter_test.cc

```diff
@@ -312,50 +211,48 @@ TEST_F(CudnnFusedMhaRewriterTestHloTest,
             2);
 }
 
-TEST_F(CudnnFusedMhaRewriterPipelineTest,
-       BF16Bmm1Bmm2Pattern_bmm1_rhs_contracting_dim_not_most_minor) {
-  if (skip_reason_) GTEST_SKIP() << *skip_reason_;
-  TF_ASSERT_OK_AND_ASSIGN(
```

Comment:
Same error, "Lines should be <= 80 columns wide [whitespace/line_length] [2]", trailing whitespace again?